### ISSUE TITLE
Not displaying right text in Thai font

### ISSUE THREAD

**Issue by bobotdls** [frappe/print_designer#100]
Not displaying right text in Thai font
It's display a box only 
![Capture](https://github.com/frappe/print_designer/assets/38277965/3a468263-98c2-4b1f-b9f1-56b554226151)

**Comment by maharshivpatel** [comment]
@bobotdls  Not really a Print designer issue as far as i know.
Fonts you are using doesn't support Thai. Can you please share the name of the font .

**Comment by bobotdls** [comment]
Bai Jamjuree & sans-serif
These two font I use in customize print format in CSS

**Comment by bobotdls** [comment]
Only on generating the pdf... If I print it's show the right font

**Comment by maharshivpatel** [comment]
> Only on generating the pdf... If I print it's show the right font

Yeah, because we don't have `Bai Jamjuree` font that is why it is showing this and on preview it is showing probably because you may have fonts installed on local machine that support Thai

**Comment by bobotdls** [comment]
after I download the said font, where do I put the file on the eprnext folder?

**Comment by maharshivpatel** [comment]
you just install it on the server/computer

**Comment by bobotdls** [comment]
ok thanks... Now it's working..
![Capture](https://github.com/frappe/print_designer/assets/38277965/44be66de-1e96-41ee-9b20-38bc7ee869cd)

**Comment by maharshivpatel** [comment]
@PhantomKunai problem is all fonts have different fontWeights and Italic fontWeights. we need this information for it to work. we can get them from google's font api but it is paid.

we can add as many fonts we want just add it to the list and raise pr i will merge it.

**Comment by PhantomKunai** [comment]
@maharshivpatel Ah I see. Perhaps one may contribute widely used fonts to the list, and maybe we can find some way to add fonts through a custom app for those who have some obscure font requirements or they are really looking to work with custom fonts.

Is my understanding correct that only these fonts are displayed in the font selection box?

**Comment by PhantomKunai** [comment]
![image](https://github.com/frappe/print_designer/assets/12585952/d0b9154e-b637-4acf-88cb-4392e65436a8)

Maybe even some way to fill those font parameters in manually somewhere here or perhaps in Print Designer Settings page, if there is one.

**Comment by j0nathanr** [comment]
I'm trying to use a custom font in a dockerized set up of Frappe Print Designer. I've edited the container file so the font is installed on the backend container.
`COPY fonts/micrenc.ttf /usr/share/fonts/truetype/micr/micrenc.ttf
RUN apt-get update && \
    apt-get install -y fontconfig && \
    fc-cache -fv && \
    rm -rf /var/lib/apt/lists/*`
 The issue I'm having is when trying to generate PDFs, I'm getting the same error as issue #242 ('NoneType' object is not iterable) as if the google fonts aren't available. If I change the print format to custom, then I get the error "Error Attaching File". Is this expected behavior in a dockerized set up? Is there no way to accomplish this as of now?
